pages/accountLists/[accountListId]/tools/mergeContacts.page.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import Head from 'next/head';
+import MergeContacts from '../../../../src/components/Tool/MergeContacts/MergeContacts';
+
+const mergeContacts: React.FC = () => {
```

Comment:
Usually React components have the first letter capitalized
```suggestion
const MergeContacts: React.FC = () => {
```